### PR TITLE
Nerfs payout for Exploration missions

### PR DESCRIPTION
**Pull request by Penwin0**
## About The Pull Request
Was not sure how to go about this so I just shaved off a zero from every payment, If there's a way in which Maintainers would rather this be done Ill change as requested.

## Why It's Good For The Game
Exploration shouldn't be flooding the station with practically unlimited funds, discovery point gain remains unaltered

## Changelog
:cl:
balance: Reduces the ammount of cash from exploration missions
/:cl:

### CODE REVIEW

**Review by Archanial — DISMISSED** [review]

**Comment by St0rmC4st3r** [comment]
Did you account that this money is shared between all the budgets?

**Comment by Penwin0** [comment]
Station should not be receiving up to 400k total credits per mission, Yes the rewards will be smaller per department but Its not exploration's role to bring money to the station, this also ended up with very unbalanced situations where threats such as blob could not compete with a cargo with infinite cash from the budgets

**Comment by OceanFish1** [comment]
> Station should not be receiving up to 400k total credits per mission, Yes the rewards will be smaller per department but Its not exploration's role to bring money to the station, this also ended up with very unbalanced situations where threats such as blob could not compete with a cargo with infinite cash from the budgets

Major skill issue for blob

**Comment by Kapu1178** [comment]
Return some of the funds to other departmental income sources. 

Having exploration be the only source of mass income is just fucking annoying.

**Comment by Penwin0** [comment]
Forgot to edit the announcement I'll fix that when I get home today

**Comment by PowerfulBacon** [comment]
They were too high before, but I think this is way too low considering the numbers get like divided by 5 for each department, so artifact recovery would be 1000, which is about 3000 credits for medbay in a shift since you can only do about 3ish missions on average per shift if you are good

**Comment by Penwin0** [comment]
Exploration isn't supposed to be a source of income, that's Cargo's job

**Comment by PowerfulBacon** [comment]
> Exploration isn't supposed to be a source of income, that's Cargo's job

Cargos job is to manage cargo and make money on the side by completing bounties for Nanotrasen. Explorations main job is to conduct research by exploring anomalies of which funds are provided by Nanotrasen to complete. Neither cargo nor exploration have the main job of making money but both are a source of income.

File: code/modules/shuttle/super_cruise/orbital_poi_generator/objective_types/recover_blackbox.dm [hunk 1]
```diff
@@ -4,8 +4,8 @@
 	//The blackbox required to recover.
 	var/obj/item/blackbox/objective/linked_blackbox
 	//Relatively easy mission.
-	min_payout = 50000	//10k credits for sci/sec/eng, 5k for ser / civ
-	max_payout = 100000	//20k credits for sci/sec/eng, 10k for serv / civ
+	min_payout = 5000	//10k credits for sci/sec/eng, 5k for ser / civ
+	max_payout = 10000	//20k credits for sci/sec/eng, 10k for serv / civ
```
Comment (KubeRoot) [inline code/modules/shuttle/super_cruise/orbital_poi_generator/objective_types/recover_blackbox.dm]:
```suggestion
	min_payout = 5000	//1k credits for sci/sec/eng, 500 for ser / civ
	max_payout = 10000	//2k credits for sci/sec/eng, 1k for serv / civ
```

Presumably the comments should also have the numbers reduced tenfold?

Comment (Penwin0) [reply code/modules/shuttle/super_cruise/orbital_poi_generator/objective_types/recover_blackbox.dm]: Yes, I'll do it when I am able to today

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**